src/ui/Button.tsx

```diff
@@ -108,7 +101,6 @@ export const Button = forwardRef<HTMLButtonElement, Props>(
         onClick={to ? () => navigate(to) : onClick}
       >
         {children}
-        {type === "secondary-download" && <IconDownload src={iconDownload} />}
         {type === "tertiary" && <IconGo src={iconGo} />}
```

Comment:
I suppose, similarly, "tertiary" is also being hardcoded to "Go" icon.  We should also do similar there, let caller passes that in.

Reply:
+1

Reply:
https://lucky-smakager-43f6a1.netlify.app/#forms-buttons-tertiary
The "Go" icon is very much part of the tertiary button. It's the only thing that marks the tertiary button as a button!